docs/source/deprecation.md

```diff
@@ -6,46 +6,41 @@ is evolving, it happens that some functionality needs to be changed, replaced
 or dropped completely. Such situations are inevitable. To reduce negative
 impact on your code, we introduce the deprecation process described below.
 
-We are still working on improvement of the current situation to provide stable
-interfaces for people, so extensibility of the current solutions could be much
-easier. This is just on of those improvements.
```

Comment:
The whole paragraph is quite hard to grasp. It's so abstract that I don't even know what is the message I should take from it. I think it can be removed all.

Reply:
the removal is ok for me